src/greeting.rs

```diff
@@ -40,7 +45,11 @@ pub fn welcome(settings: &Settings) {
     let ui_link = format!(
         "http{}://{}:{}/dashboard",
         if settings.service.enable_tls { "s" } else { "" },
-        settings.service.host,
+        if is_zero_ip(&settings.service.host) {
+            "localhost"
```

Comment:
Localhost is `127.0.0.1`, you may check not zero ip

Reply:
I've patched this now:

These are `localhost` now:
- `0.0.0.0`
- `::0` (or equivalent)
- `127.0.0.1`
- `::1` (or equivalent) on macOS, because on Linux this is `ip6-localhost`